### PR TITLE
Move templates to RHEL/CentOS/UBI 8

### PR DESCRIPTION
**Pull request by yselkowitz**
This has already been tested in image ecosystem:

https://github.com/openshift/cluster-samples-operator/pull/319

### CODE REVIEW

**Review by frenzymadness — APPROVED** [review]
The change looks good to me. It should be merged also to 2.2.x branch.

File: openshift/templates/django-postgresql.json [hunk 1]
```diff
@@ -421,15 +421,15 @@
...
+      "value": "3.6-ubi8",
       "required": true
     },
     {
       "name": "POSTGRESQL_VERSION",
       "displayName": "Version of PostgreSQL Image",
-      "description": "Version of PostgreSQL image to be used (10 or latest).",
-      "value": "10",
+      "description": "Version of PostgreSQL image to be used (10-el7, 10-el8, or latest).",
+      "value": "10-el8",
```
Comment (hhorak) [inline openshift/templates/django-postgresql.json:432]: We can use `12-el8` here already.

**Comment by hhorak** [comment]
Just a comment about postgresql version, otherwise LGTM.

**Comment by hhorak** [comment]
Hm, one more thought -- since branch `2.2.x` exists in this repo and can be used to have a newer Django for RHEL 8 containers (RHEL-8 only because Django 2+ requires a newer sqlite than what is in RHEL-7), shouldn't the template defaults to this branch? It means the template parameter `SOURCE_REPOSITORY_REF` could point to `2.2.x` by default.
I'm personally not 100% sure whether this is a correct thing to do, and we don't need to block this PR on this. I just wanted to bring this up here. It might be reported and discussed separately.

**Comment by yselkowitz** [comment]
Changes incorporated, please re-review.

**Comment by hhorak** [comment]
Perfect, thanks a lot. Merging.

**Pull request merged**